state_management/csharp/sdk/order-processor/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Dapr.Client;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading;
+using System.Text.Json;
+using System.Text;
+using System.Text.Json.Serialization;
+
+string DAPR_STORE_NAME = "statestore";
+while(true) {
+    System.Threading.Thread.Sleep(5000);

```

Comment:
I think it functions and reads a little better to put the Thread sleep at the very bottom of while loop.

Suggest this code instead too:

    `await Task.Delay(TimeSpan.FromSeconds(5));`